src/app/services/fiddle.service.spec.ts

```diff
@@ -1,11 +1,30 @@
 import {FiddleService} from './fiddle.service';
+import {Http, HTTP_PROVIDERS, Response} from 'angular2/http';
+import {  
+    expect, 
+    it, 
+    describe,
+    injectAsync,
+    TestComponentBuilder,
+    beforeEachProvider
```

Comment:
Ahhh. Found it while typing up the response. This is a simple typo. Needs an s at the end.